src/dmd/statementsem.d

```diff
@@ -1223,19 +1219,118 @@ private extern (C++) final class StatementSemanticVisitor : Visitor
             }
         }
 
-        Statement s;
+        void retError()
+        {
+            sc2.pop();
+            result = new ErrorStatement();
+        }
```

Comment:
add newline

Reply:
done

Reply:
There are multiple places with missing newlines.
I didn't want to point out every single one ;)